projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -374,7 +374,7 @@ public function post_page_metabox_assets() {
 	$( document ).ready( function() {
 		// If we have the #pub-connection-tests div present, kick off the connection test
 		if ( $( '#pub-connection-tests' ).length ) {
-			publicizeConnTestStart();
+			// publicizeConnTestStart(); TODO: Fix magic links
```

Comment:
What's involved with fixing this? Is it something we should do now?

Reply:
Well, probably not. I think it needs more investigation beforehand, something is up with the logic of that. I would rather leave this as is for now and fix this with the design update together in the next task. What do you think?

Reply:
Yeah makes sense!